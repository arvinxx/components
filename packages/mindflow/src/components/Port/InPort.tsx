import React from 'react';
import type { FC } from 'react';

import './style.less';

interface InPortProps {
  color: string;
}

export const InPort: FC<InPortProps> = ({ color }) => {
  return (
    <div
      style={{
        borderColor: color,
      }}
      className="mind-port-in-port"
    />
  );
};
