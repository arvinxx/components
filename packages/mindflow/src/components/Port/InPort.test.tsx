import React from 'react';
import { render } from '@testing-library/react';

import { InPort } from './InPort';

describe('InPort', () => {
  it('默认', () => {
    const { container } = render(<InPort color={'red'} />);
    expect(container).toMatchSnapshot();
  });
});
