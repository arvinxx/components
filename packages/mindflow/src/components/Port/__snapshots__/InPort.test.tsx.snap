// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InPort 默认 1`] = `
<div>
  <div
    class="mind-port-in-port"
    style="border-color: red;"
  />
</div>
`;
